### PR TITLE
Blog post about "Cancel old builds" feature

### PR DESCRIPTION
**Pull request by humitos**
Initial content for the blog post.

### CODE REVIEW

File: cancel-old-builds.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   :tags: builds, ci
+   :author: Manuel
+   :location: BCN
+
+.. meta::
+   :description lang=en:
+      A new feature that auto-cancels invalid builds was deployed. It improves the user experience,
+      reduces resource costs and also energy waste.
+
+Auto-cancel invalid builds
```
Comment (ericholscher) [inline cancel-old-builds.rst:11]: This is a weird title that leaves me with multiple questions. Who is cancelling builds? What is an invalid build? etc. 

```suggestion
Platform update: Auto-canceling builds when pushing to the same branch twice
```

File: cancel-old-builds.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+.. image:: /img/cancelled-builds.png
+   :align: center
+   :width: 100%
+
+
+Today, we are happy to announce that we decided to enable this feature for all the projects because we have received really good feedback about it
+and we noticed a pretty good improvement on the user experience when triggering multiple builds frequently.
+
+Thanks you all for you help while working on this feature and we hope you get a better experience starting today.
+Let us know if you find anything not working as expected.
```
Comment (ericholscher) [inline cancel-old-builds.rst:38]:
```suggestion
Today, we are happy to announce that we are enabling this feature for all the projects.
We have received really good feedback from our test users,
particularly around the user experience when triggering multiple builds frequently.

We appreciate the feedback that our test users gave us,
and we're excited to roll this out to all our users.
Please let us know if you any new issues around cancelled builds as we roll out this feature more widely.
```

**Review by ericholscher — COMMENTED** [review]
This is good to announce, I did some basic copy editing. I'm not 100% sure w ewant to include each project name on this graph -- it seems oddly specific. I think a more useful graph is just cancelled builds per day? But overall I'm not sure what we're trying to communicate with the graph?

**Comment by humitos** [comment]
> I think a more useful graph is just cancelled builds per day? But overall I'm not sure what we're trying to communicate with the graph?

Good point. I think cancelled builds per day works better, yes. The message this graph communicates in this context is mainly related to the improvement of the UX because each build cancelled means a user didn't have to wait for an invalid build to finish. Also, it's related to the reduction of the costs and energy saving. I'm going to change the plot for "cancelled builds per day".

File: cancel-old-builds.rst (hunk 1 above)
Comment (humitos) [reply cancel-old-builds.rst:11]: 👍🏼 --I removed the "Platform update" part because I think it's not needed and left the rest.

**Comment by humitos** [comment]
I updated the blog post. Let me know if it's ready for you: https://readthedocs-blog--182.org.readthedocs.build/cancel-old-builds/

**Comment by ericholscher** [comment]
Looks good. I'll be curious what this graph looks like next week. Could be a neat tweet followup.

**Review by ericholscher — APPROVED** [review]

**Pull request merged**